### ISSUE TITLE
Is it possible to backpropagate trough eigan vectors?

### ISSUE THREAD

**Issue by Jeevi10** [liyu10000/DeepLDA#2]

**Comment by liyu10000** [comment]
Backpropagate needs to start from a single loss. If you want to get good eigen vectors, you may want to design some metrics to enforce this. For example, if you want two images get similar eigen vectors, you can use Euclidean distance or cosine distance on them as a loss function and do the backpropagation.
